recipe/meta.yaml

```
@@ -1,48 +1,63 @@
 {% set name = "xmlsec" %}
-{% set version = "1.3.14" %}
+{% set version = "1.3.17" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 934f804f2f895bcdb86f1eaee236b661013560ee69ec108d29cdd6e5f292a2d9
+  url: https://github.com/xmlsec/python-xmlsec/archive/refs/tags/1.3.17.tar.gz
+  sha256: 71967b9fbde59690d7e1a6b37d79f30e006d1fd3060b2a3e09e6472f454f4a99
 
 build:
   number: 0
-  # Skip s390x because of missing libxmlsec1, libgpg-error and libgcrypt
-  skip: true  # [(linux and s390x) or win or py<35]
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: true  # [py<39]
+  # libxmlsec1 is not in the main channel for Windows
+  skip: true  # [win]
 
 requirements:
   build:
+    - {{ stdlib('c') }}
     - {{ compiler('c') }}
     - pkg-config
   host:
-    - python
-    - libxml2 {{ libxml2 }}
-    - libxmlsec1 1.3.5
-    - lxml 5.2.1
     - pip
-    - pkgconfig
-    - setuptools
-    - setuptools_scm
-    - toml
+    - python
+    - setuptools >=80.9.0
     - wheel
+    - setuptools_scm >=3.4
+    - toml
+    - pkgconfig >=1.5.1
+    # Used the same as in the requirements.txt and it was used for tests in the upstream -> https://github.com/xmlsec/python-xmlsec/actions/runs/19269549253/job/55093915102#step:5:21
+    - lxml ==6.0.2
+    # From readme -> https://github.com/xmlsec/python-xmlsec/blob/1.3.17/README.md#requirements
+    - libxml2 >=2.9.1
+    - libxmlsec1 >=1.2.33
   run:
     - python
-    - lxml >=3.8.0
+    # https://github.com/xmlsec/python-xmlsec/blob/master/requirements.txt#L1
+    - lxml ==6.0.2
```

https://github.com/xmlsec/python-xmlsec/blob/1.3.17/setup.py#L27
```suggestion
    - lxml >=3.8
```